functions/find-type.py

```diff
@@ -0,0 +1,6 @@
+def find_type_arg(value): # Fuction to find type of given value.	
+	print type(value)
```

Comment:
Use function doc string based on this reference https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html